src/Steam.Common/GameId.cs

```diff
@@ -0,0 +1,188 @@
+using System;
+
+namespace Steam
+{
+    /// <summary>
+    /// Definition of a GameId, interpreted from a <see cref="ulong"/> value
```

Comment:
Leaking implementation details ("from a ulong value"). Just talk about what the game ID actually is and what it's for.

Reply:
In this case a Game ID is a unique identifier for a game on the Steam network.